app/assets/stylesheets/transactions.css.scss.erb

```diff
@@ -0,0 +1,20 @@
+#new_transfer_form {
+  display: none;
+}
+
+.exchange-helper {
+  font-size: 80%;
+}
+#total_balance {
+  .totals {
+    margin-bottom: 15px;
+  }
+}
+
+.positive.new-transaction {
+  background-color: darken(<%= Dictionaries.account_colors['positive'] %>, 20) !important;
```

Comment:
@yunixon create file `variables.css.scss` and create there variable. use this variable here. Don't use ruby code in css files.